index.html

```diff
@@ -27,58 +27,59 @@
                 <li><a href="#about-me">About</a></li>
                 <li><a href="#form-start">Contact</a></li>
             </ul>
-            <button class="only-mobile">
+            <button class="only-mobile menu-toggle" type="button"> 
                 <svg xmlns="http://www.w3.org/2000/svg" class="icon-md" fill="none" viewBox="0 0 24 24"
-                    stroke="currentColor">
-                    <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M4 6h16M4 12h16M4 18h16" />
-                </svg>
-            </button>
-        </nav>
-    </header>
-    <main >
-        <section id="landing">
-
-            <div class="landiv">
-                <h1>Hey there. I'm Abel<br><span class="greeny">I'm a software developer</span></h1>
-                <p id="main-support-text"> I can help you build a product , feature or website Look through some of my work
-                    and experience! If you like what you see and have a project you need coded,
-                    don’t hestiate to contact me.</p>
-                </div>
-                <div class="logo-cloud">
-                    <ul>
-                        <li>
-                            <a href="#"><img src="./assets/svg/twitter.svg" alt="link to abel's Twitter profile"></a>
-                        </li>
-                        <li>
-                            <a href="#"><img src="./assets/svg/Linkedin-icon.svg" alt="link to abel's LinkedIn"></a>
-                        </li>
-                        <li>
-                            <a href="#"><img src="./assets/svg/medium.svg" alt="link to abel's Medium"></a>
-                        </li>
-                        <li>
-                            <a href="#"><img src="./assets/svg/github.svg" alt="link to abel's Github"></a>
-                        </li>
-                        <li>
-                            <a href="#"><img src="./assets/svg/angellist.svg" alt="link to abel's Angellist"></a>
-                        </li>
-                    </ul>
-                </div>
-                <div class="scroll-down">
-                    <a href="#my-work">
-                        <img src="./assets/png/arrowtip.png" alt="button to works section"> 
-                    </a>
-                </div>
-            </section>
-                
-                <section id="my-work">
-                    <div class="first section">
-                <h2>My Recent Works</h2>
-                <img src="./assets/png/separator.png" alt="separator image">
+                stroke="currentColor">
+                <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M4 6h16M4 12h16M4 18h16" />
+            </svg>
+        </button>
+    </nav>
```

Comment:
- [ ] For the mobile version the navbar when is displayed is overlaying the whole height of the website and the user can scroll down and only will see a white screen.
![Captura de pantalla_2021-10-04_19-09-48](https://user-images.githubusercontent.com/53111464/135940787-6b07ba10-a40d-4da9-9ccb-6e61e0c30b78.png)
-  It's not necessary that the navbar overlay the whole website instead you can use a position sticky property and make this container stick the screen to show the nav items.- [ ] For the mobile version the navbar when is displayed is overlaying the whole height of the website and the user can scroll down and only will see a white screen.
![Captura de pantalla_2021-10-04_19-09-48](https://user-images.githubusercontent.com/53111464/135940787-6b07ba10-a40d-4da9-9ccb-6e61e0c30b78.png)
-  It's not necessary that the navbar overlay the whole website instead you can use a position sticky property and make this container stick the screen to show the nav items.

Reply:
hey @Alexoid1 I checked what you said but I can't figure out why it appears like this to you, the position is actually fixed so is not scrollable, and it appears correctly on my device, is there a way you could show me what's happening? I don't know how to implement the changes since I can't reproduce that issue.

Thanks for your feedback!